src/components/decision/verification/css/style.css

```diff
@@ -34,6 +34,14 @@
   }
 }
 
+.decision-group {
+  /*
+    By using box-decoration-break, it will round the edges if the content
+    wraps.
+  */
+  box-decoration-break: clone;
```

Comment:
i don't think we ever want a group to split. will a group ever be smaller than screen width?

Reply:
Valid and fair